cpp/src/parquet/properties.h

```diff
@@ -525,7 +525,8 @@ class PARQUET_EXPORT WriterProperties {
     /// Enable writing page index in general for all columns. Default disabled.
     ///
     /// Page index contains statistics for data pages and can be used to skip pages
-    /// when scanning data in ordered and unordered columns.
+    /// when scanning data in ordered and unordered columns. Note that it does not
```

Comment:
What is "it" in this context? The sentence isn't very clear.

Reply:
Here "it" denotes to the parquet writer or column writer.

Reply:
Perhaps:
> Writing statistics to the page index disables the old method of writing statistics to each data page header.
> The page index makes filtering more efficient than the page header, as it gathers all the statistics for a Parquet file in a single place, avoiding scattered I/O.